src/NzbDrone.Core/Download/Clients/Nzbget/Nzbget.cs

```diff
@@ -161,7 +161,7 @@ private IEnumerable<DownloadClientItem> GetHistory()
 
                 if (!_successStatus.Contains(item.DeleteStatus) && item.DeleteStatus.IsNotNullOrWhiteSpace())
                 {
-                    if (_deleteFailedStatus.Contains(item.DeleteStatus))
+                    if (_deleteFailedStatus.Contains(item.DeleteStatus) || _deleteFailedStatus.Contains(item.MarkStatus))
```

Comment:
This block has nothing to do with `MarkStatus`, it should not be responsible for marking a manually deleted item with a mark status of `BAD` as failed, same as it doesn't mark failures due to unpacking or scripts as failed here.

Reply:
Requested changes have been made. Regarding tests, it might take me a few more days to find the time to do that.